### PR TITLE
[TEST] Billing Service REST

### PR DESCRIPTION
**Pull request by stevenzeiler**
Add HTTP Tests for the Billing Service REST endpoints

### CODE REVIEW

File: test/http/billing_service.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  var token
+
+  before(function(done) {
+    new codius.Token({ token: uuid.v4() }).save()
+      .then(function(model) {
+        token = model
+        done()
+      })
+  })
+
```
Comment (wilsonianb) [inline test/http/billing_service.js]: Should there be an `after` that deletes the token?

Comment (stevenzeiler) [inline test/http/billing_service.js]: Only if you aren't cleaning out your test database after each run. We would also have to delete the associated `balance`, and and `credit` and `debit` records

Comment (stevenzeiler) [inline test/http/billing_service.js]: Gatewayd solves this problem by using sqlite in-memory for tests, rather than persisting data.

**Comment by wilsonianb** [comment]
LGTM :+1:

**Comment by codius-montecarlo** [comment]
The build is good and I'm seeing 1 +1s from the following folks:

* wilsonianb

As such, I'll attempt to merge and close this pull request.

**Pull request merged**